python/tests/integration/test_async_dispatch_server_integration.py

```
@@ -59,37 +59,7 @@ def test_dispatch_probes_in_process():
 class _EchoHandler(socketserver.StreamRequestHandler):
     def handle(self):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `handle`

Add return type annotation: `None`

(ANN202)